app/src/main/java/com/example/scanpal/MainActivity.java

```diff
@@ -1,8 +1,12 @@
 package com.example.scanpal;
 
 import android.os.Bundle;
+import android.view.ViewGroup;
```

Comment:
are these imports required?

Reply:
Nope! ill get rid of that, it was a remnant from when I was testing something else.